src/js/utils/apiClient.jsx

```diff
@@ -9,13 +9,13 @@ import notification from 'components/Layout/notifications/notification';
 import LoginModal from 'components/LoginModal';
 import NotificationType from 'consts/notificationTypes';
 
-const justRejectRequestError = error => Promise.reject(error);
+const justRejectRequestError = (error) => Promise.reject(error);
```

Comment:
seems like ESLint rules have changed that he shouted for this? Or have you done it manually?

Reply:
Yes, I used fix all eslint problems when I was working on this file and it added those brackets.